### PR TITLE
Additional considerations in Production

### PR DESCRIPTION
**Pull request by CarlSargunar**
Added a suggestion to stop IIS while upgrading/deploying a v9 site to production, as I came across this myself when upgrading.

### CODE REVIEW

**Comment by eshanrnh** [comment]
Thanks for the PR, @CarlSargunar 🙌 We will review this soon.

**Comment by sofietoft** [comment]
Thanks again for the PR @CarlSargunar ! 💪 
I think it makes a lot of sense to add a note about this to the guide!

I do have a couple of questions to clear things up:
- It seems like this sentence is unfinished: "The new files will be re-created when you deploy, and this way there's little chance of the." Could you help me understand what is missing here?
- Adding this note after the steps, might not help the ones who've already followed the steps. How about we add the note above the steps? Or perhaps even add "Stop the site in IIS" as a step entirely? 🤔 

Finally, I realize that it might sometimes be necessary, but we shouldn't recommend people to upgrade in Production. Perhaps a little note about that as well?

Hope it all makes sense, otherwise do let me know!

**Comment by CarlSargunar** [comment]
Haha - oops! I finished the PR on a different laptop, but looks like I didn't make all the changes - I'll update this and re-submit it, but I also will make a note about upgrading in production too (with an appropriate warning)

**Comment by sofietoft** [comment]
Ah, no worries @CarlSargunar ! 😄 
I'll pop back in to check a little later on then!

**Comment by CarlSargunar** [comment]
OK - re-structured it a little based on the comments, and added the rest of my missing sentence. Thanks :-)

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 1]
```diff
@@ -78,9 +81,9 @@ The following options may suit your needs:
     - `/umbraco/config/lang`
     - `/umbraco/config/appsettings-schema.json`
 
-10. If using Umbraco Forms, update your files and folders according to the [Upgrading - version specific](../../../Add-ons/UmbracoForms/Installation/Version-Specific.md) for version 10 article.
+11. If using Umbraco Forms, update your files and folders according to the [Upgrading - version specific](../../../Add-ons/UmbracoForms/Installation/Version-Specific.md) for version 10 article.
 
-11. Build and run your project to finish the installation of Umbraco 10.
+12. Re-start your site in IIS, build and run your project to finish the installation of Umbraco 10.
```
Comment (sofietoft) [inline Fundamentals/Setup/Upgrading/version-specific.md:86]:
```suggestion
12. Restart your site in IIS, build and run your project to finish the installation of Umbraco 10.
```

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 2]
```diff
@@ -94,10 +97,31 @@ To:
...
+
 :::note
 To upgrade to Umbraco 10, your database needs to be at least on Umbraco 8.18.
 :::
 
+### Upgrade of any publicly hosted environment
+
+When the upgrade is completed and tested, and prior to deploying to any publicly accessible environment, you should consider the following:
+
+1. You will want to ensure you have backups for both the datbaase and the file system for rollback.
```
Comment (sofietoft) [inline Fundamentals/Setup/Upgrading/version-specific.md:109]:
```suggestion
1. Ensure you have backups for both the database and the file system.
```

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 3]
```diff
@@ -94,10 +97,31 @@ To:
...
+2. You should stop the site so it is not accessable during the upgrade process.
+3. You should delete the relevant folders from the filesystem prior to deploying.
```
Comment (sofietoft) [inline Fundamentals/Setup/Upgrading/version-specific.md:111]:
```suggestion
2. Stop the site so it is not accessible during the upgrade process.
3. Delete the relevant folders from the filesystem prior to deploying.
```

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 4]
```diff
@@ -94,10 +97,31 @@ To:
...
+  - `/wwwroot/umbraco`
+  - `/umbraco/PartialViewMacros`
+  - `/umbraco/UmbracoBackOffice`
+  - `/umbraco/UmbracoInstall`
+  - `/umbraco/UmbracoWebsite`
+  - `/umbraco/config/lang`
+  - `/umbraco/config/appsettings-schema.json`
+4. Deploy the site how you normally would to your public facing environment 
+5. Start the site. At this point it will launch and upgrade the database, after which the site should become accessible and your upgrade is complete. 
+6. Remember to check the logs for any errors which may have occurred during the upgrade process.
```
Comment (sofietoft) [inline Fundamentals/Setup/Upgrading/version-specific.md:121]:
```suggestion
6. Check the logs for any errors which may have occurred during the upgrade process. 
```

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 5]
```diff
@@ -94,10 +97,31 @@ To:
...
+  - `/umbraco/PartialViewMacros`
+  - `/umbraco/UmbracoBackOffice`
+  - `/umbraco/UmbracoInstall`
+  - `/umbraco/UmbracoWebsite`
+  - `/umbraco/config/lang`
+  - `/umbraco/config/appsettings-schema.json`
+4. Deploy the site how you normally would to your public facing environment 
+5. Start the site. At this point it will launch and upgrade the database, after which the site should become accessible and your upgrade is complete. 
+6. Remember to check the logs for any errors which may have occurred during the upgrade process. 
+:::
```
Comment (sofietoft) [inline Fundamentals/Setup/Upgrading/version-specific.md:122]: This `:::` is closing a note. But I don't see it started anywhere? 🤔 

Did you mean to put this part in a note/tip, or should these three colons be taken out?

**Review by sofietoft — COMMENTED** [review]
Hi @CarlSargunar !

Thanks for updating the PR 💪 😄 
I've made a couple of suggestions. Mainly to the steps as we prefer starting each item in a list with the action word. Hope it all makes sense 🤞

**Comment by sofietoft** [comment]
Looks great @CarlSargunar ! 😄 

Thanks again for updating this article 💪 
Also, it's Hacktoberfest, and this contribution [counts toward the challenge](https://umbraco.com/blog/umbraco-hacktoberfest-2022/) 🥳 
I'll get it merged in!

**Pull request merged**